output/.gitignore

```diff
@@ -1,2 +1,6 @@
 *
-!.gitignore
\ No newline at end of file
+!.gitignore
```

Comment:
probably to delete this file too :)

Reply:
wasn't aware of the second gitignore. It has been deleted.